lib/order.rb

```diff
@@ -0,0 +1,49 @@
+require_relative 'menu'
+
+class Order
+
+  attr_reader :complete_order, :menu, :order_total
+
+  def initialize
+    @menu = Menu.new
+    @complete_order = {}
+    @order_total = 0
+  end
+
+  def add(item, quantity)
+    fail "Not on menu" unless menu.menu_list.key?(item)
+    update_price(item, quantity)
+    add_to_order(item, quantity)
+  end
+
+  def display_menu
+    menu.menu_list.each { |key, value| puts "#{key} - £#{value}" }
+  end
+
+  def receipt
+    complete_order.map do |item, quantity|
+      "#{quantity}x #{item} - £#{menu.menu_list[item] * quantity}"
+    end.join("\n")
+  end
+
+  def tax_receipt
+    return receipt + "\nTax @8.64%      £#{(order_total * 0.0867).round(2)} \n" +
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/LineEndConcatenation: Use `\` instead of `+` or `<<` to concatenate those strings.<br>Style/RedundantReturn: Redundant `return` detected. (https://github.com/bbatsov/ruby-style-guide#no-explicit-return)